cypress/page/shopping-cart.page.ts

```diff
@@ -3,7 +3,8 @@ class ShoppingCartPage {
     private goToCheckoutBtn: string;
 
     constructor() {
-        this.goToCartBtn = "[style*='display: block;'] .button-container > a";
+
+        this.goToCartBtn = ".button-container > .button-medium";
         this.goToCheckoutBtn = ".cart_navigation span";
```

Comment:
this can be improved using the title property to specify we are looking for the `proceed to checkout` button